src/Chartjs.vue

```diff
@@ -49,7 +45,7 @@ export default {
       this.$nextTick(() => {
         // this.chart.data.datasets = val.datasets
         // this.chart.data.labels = val.labels
-        this.chart.update()
+        this.mountChart()
       })
```

Comment:
@fundon I thank you for the good work.
About the question, with this condition the error disappears, but the canvas doesn't draw